src/stylesheets/components/_icon.scss

```diff
@@ -71,3 +71,7 @@ sds-icon.external-link > * > * {
     display: block;
   }
 }
+
+usa-icon.text-white {
```

Comment:
.text-white is already a USWDS class applying #ffffff.

Reply:
It's in the .postfix class search.component.scss